src/client/utils/browser.js

```diff
@@ -1,9 +1,3 @@
-function getMSEdgeVersion (userAgent) {
-    var edgeStrIndex = userAgent.indexOf('edge/');
-
-    return parseInt(userAgent.substring(edgeStrIndex + 5, userAgent.indexOf('.', edgeStrIndex)), 10);
-}
-
 function calculateBrowserAndVersion (userAgent) {
     var webkitRegEx  = /(webkit)[ \/]([\w.]+)/;
```

Comment:
Sometimes it returns version of browser and sometimes version of engine. I don't really like it